src/panda.cpp

```diff
@@ -14,6 +14,8 @@
 
 #include <RBDyn/parsers/urdf.h>
 
+#include <filesystem>
```

Comment:
This breaks packaging on 18.04

Can you switch to boost/filesystem instead?

Reply:
Too bad I used `std::filesystem` on purpose :P
Fixed.